JavaScript/JavaScriptSDK.Tests/CheckinTests/Telemetry/PageViewPerformance.tests.ts

```diff
@@ -90,12 +90,12 @@ class PageViewPerformanceTelemetryTests extends ContractTestHelper {
             test: () => {
 
                 var timing = <PerformanceTiming>{};
-                timing.navigationStart = 1;
-                timing.connectEnd = 10;
-                timing.requestStart = 11;
-                timing.responseStart = 30;
-                timing.responseEnd = 42;
-                timing.loadEventEnd = 60;
+                (<any>timing).navigationStart = 1;
```

Comment:
PerformanceTiming properties are read-only, to change them I need to cast them to _any_. 
One way to fix this is to remove PerformanceTiming type and create a simple object with required properties. I like strongly typed objects, but in this case casting to _any_ contradicts using a type, doesn't it? 

```
                var timing = {
                    navigationStart: 1,
                    connectEnd: 10,
                    requestStart: 11,
                    responseStart: 30,
                    responseEnd: 42,
                    loadEventEnd: 60,
                };
```
instead of 
```
                var timing = <PerformanceTiming>{};
                (<any>timing).navigationStart = 1;
                (<any>timing).connectEnd = 10;
                (<any>timing).requestStart = 11;
                (<any>timing).responseStart = 30;
                (<any>timing).responseEnd = 42;
                (<any>timing).loadEventEnd = 60;
```